tests/rkt_error_output_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+func TestRktEnsureErrors(t *testing.T) {
+	const imgName = "rkt-ensure-errors-test"
+
+	image := patchTestACI(fmt.Sprintf("%s.aci", imgName), fmt.Sprintf("--name=%s", imgName))
+	defer os.Remove(image)
+
+	ctx := testutils.NewRktRunCtx()
+	defer ctx.Cleanup()
+
+	// Prepare image
+	cmd := fmt.Sprintf("%s --insecure-options=image prepare %s", ctx.Cmd(), image)
+	_ = runRktAndGetUUID(t, cmd)
+
+	tmpDir := createTempDirOrPanic(imgName)
+	defer os.RemoveAll(tmpDir)
+
+	// Define tests
+	tests := []struct {
+		cmd    string
+		expect string
+	}{
+		// Test that stage1 is printing error output.
+		{
+			"run --insecure-options=image --net=notavalidnetwork",
+			".*stage1: failed to setup network",
+		},
+	}
+
+	// Run tests
+	for i, tt := range tests {
+		runCmd := fmt.Sprintf("%s %s %s", ctx.Cmd(), tt.cmd, image)
+		t.Logf("Running test #%d, %s", i, runCmd)
+		runRktAndCheckRegexOutput(t, runCmd, tt.expect)
```

Comment:
Since there is only one test, I'd just do 
```
runRktAndCheckRegexOutput(t, fmt.Sprintf("%s %s %s", ctx.Cmd(), tt.cmd, image), "stage1: failed to setup network")
```

So it would be about 15 lines less of code ;)